docs/sources/reference/components/beyla/beyla.ebpf.md

```diff
@@ -156,6 +166,40 @@ Name               | Type           | Description
 `owner_name`       | `string`       | Regular expression of Kubernetes owners of running Pods to match.                                           | `""`    | no
 `pod_labels`       | `map(string)`  | Key-value pairs of labels with keys matching Kubernetes Pods with the provided value as regular expression. |  `{}`   | no
 
+### prometheus block
+
+This block configures which metrics Beyla collects.
```

Comment:
Does the `prometheus` block also have an impact on the OTLP metrics?

Reply:
Actually yes, as Alloy uses the internal Beyla prometheus for collecting metrics, whichever format they are exported. Should we rename this block?